diff -up ./mozilla/security/nss/lib/freebl/intel-aes.s.710298 ./mozilla/security/nss/lib/freebl/intel-aes.s
--- ./mozilla/security/nss/lib/freebl/intel-aes.s.710298	2011-06-14 16:12:37.167353550 -0700
+++ ./mozilla/security/nss/lib/freebl/intel-aes.s	2011-06-14 16:14:14.870654107 -0700
@@ -1654,8 +1654,8 @@ intel_aes_decrypt_cbc_256:
 	.byte 0x66,0x41,0x0f,0x38,0xdf,0xc8	/* aesdeclast %xmm8, %xmm1 */
 	movdqu	112(%rdi), %xmm8
 	pxor	%xmm0, %xmm1
-	movdqu	%xmm1, (%rsi, %rax)
-	movdqu	(%r8, %rax), %xmm0
+	movdqu	(%r8, %rax), %xmm0  /* fetch the IV before we store the block */
+	movdqu	%xmm1, (%rsi, %rax) /* in case input buf = output buf */
 	addq	$16, %rax
 	cmpq	%rax, %r9
 	jne	4b
